docker/monitoring/docker-compose.yaml

```diff
@@ -53,12 +53,15 @@ services:
     container_name: grafana
     image: grafana/grafana:main
     volumes:
-      - ./grafana.ini:/etc/grafana/grafana.ini
-      - ./grafana-datasources.yaml:/etc/grafana/provisioning/datasources/datasources.yaml
-    #environment:
+      - ./grafana/config.ini:/etc/grafana/config.ini
```

Comment:
Does this work? I thought we need to have `/etc/grafana/grafana.ini`.

Reply:
Yeah, this is fine. I discovered that the only thing that is important is where the file is located, and what file extension it has. As long as those two things are right, Grafana will read the file and determine what it is.